x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -20,16 +20,35 @@ import {
   TrustedApp,
 } from '../../../../../common/endpoint/types';
 import { HttpFetchOptions } from 'kibana/public';
-import { TRUSTED_APPS_LIST_API } from '../../../../../common/endpoint/constants';
+import {
+  TRUSTED_APPS_GET_API,
+  TRUSTED_APPS_LIST_API,
+} from '../../../../../common/endpoint/constants';
+import {
+  GetPackagePoliciesResponse,
+  PACKAGE_POLICY_API_ROUTES,
+} from '../../../../../../fleet/common';
+import { EndpointDocGenerator } from '../../../../../common/endpoint/generate_data';
+import { isFailedResourceState, isLoadedResourceState } from '../state';
+import { forceHTMLElementOffsetWidth } from './components/effected_policy_select/test_utils';
+import { resolvePathVariables } from '../service/utils';
+import { toUpdateTrustedApp } from '../../../../../common/endpoint/service/trusted_apps/to_update_trusted_app';
+import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
 
 jest.mock('@elastic/eui/lib/services/accessibility/html_id_generator', () => ({
   htmlIdGenerator: () => () => 'mockId',
 }));
 
+// TODO: remove this mock when feature flag is removed
```

Comment:
Also - we probably should crate a new `describe()` section below that does at lest 2 other tests - one to check that the form looks as expected with the feature off and another that it looks as epxected with feature on.
I'm not a fan of using snapshots, but in this case, it probably makes sense since that `describe()` would be temporary until we remove the feature flag.

Reply:
Sounds good to me